src/goto-symex/symex_function_call.cpp

```diff
@@ -9,6 +9,9 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <iostream>
 #include <sstream>
 #include <cassert>
+#ifdef DEBUG
+#include <iostream>
+#endif
```

Comment:
There already is an unconditional `#include <iostream>` above - I suppose that should go away?